draft-ietf-rats-corim.md

```diff
@@ -500,35 +501,35 @@ The following describes each child item of this group.
 
 # Concise Module Identifier (CoMID) {#sec-comid}
 
-A CoMID tag contains information about hardware, firmware, or module composition that is provisioned
-in a Verifier. Each CoMID has a unique identifier (tag-id) that is used to unambigously identify
-tag instances. The tag-id may be used to cross reference CoMID tags from other manifest files (CoRIMs)
-and also within the same manifest, such as CoBOM, CoSWID, or TA-Store.
-
-A CoMID defines several types of Claims, using triples. A triple is a statement linking one object
-(subject) to another object (object) or a literal, via a predicate.
-
-Each triple defines a specific type of Claim, as given below.
-
-* Reference Triples contain Claims for Reference Values that are expected to match the attested Evidence.
-
-* Endorsed Triples contain Claims for Endorsements. These are assertions about the module that do not
-require Evidence matching to be accepted by the Verifier. Specific examples include testing or
-certification data pertaining to a module, that reflect immutable and dynamic properties of
-the module that assists Verifiers to infer more information about the Attester.
-
-* Identity Triples contain Identity Claims, that contain cryptographic credentials used for authentication.
-
-* Attestation Key Triples contain cryptographic keys that are used to verify the integrity protection
-on the Evidence received from the Attester.
-
-* Domain depndency Triples describe trust relationships between domains.
-
-* Membership Triples describe topological relationships between (sub-)modules. For example a
-composite Attester may comprise of multiple sub-Attesters (sub-modules). This triple
-defines the topological relationship between lead and sub Attester environments.
-
-* CoSWID Triples associate modules with existing CoSWID tags.
+A CoMID tag contains information about hardware, firmware, or module composition. Each
+CoMID has a unique identifier (tag-id) that is used to unambigously identify CoMID instances
+when cross referencing CoMID tags, from example in typed link relations, or in a CoBOM tag.
+
+A CoMID defines several types of Claims, using triples semantics. At a high level, a
+triple is a statement linking a subject to an object via a predicate.
+CoMID triples typically encode assertions made by the CoRIM author about Attesting or Target
+Environments and their security features, for example measurements, cryptographic key material,
+etc.
+
+The set of triple is extensible.
+The following triples are currently defined.
+
+* Reference Values triples: containing Reference Values that are expected to match Evidence
```

Comment:
" * `reference-value-triple-record`: contains Reference Values that are..."

and subsequent bullets should follow the convention of naming the CDDL triple record IMHO

Reply:
> " * `reference-value-triple-record`: contains Reference Values that are..."
> 
> and subsequent bullets should follow the convention of naming the CDDL triple record IMHO

We are linking every triple description to the section about that triple.

This is just a top level introduction section. The Triples section should dive into exact names of the triples in my view!

Lets discuss this aspect in the meeting tomorrow!